Codecademy/gamut
# fix(GlobalHeader): a11y fixes for resources & catalog drop downs

### Overview



1. hides decorative right arrow icon from screen readers on links (in resources dropdown, the arrow next to 'view all docs' and 'view all topics'; in the catalog dropdown it's next to 'take our quiz')
2. hides decorative box svg from catalog dropdown (next to "dont know where to start" in catalog dropdown)



### PR Checklist

- [x] Related to JIRA ticket: [DISC-898], [DISC-900]

#### Testing Instructions



[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/33088)  | [Monolith Env](https://pr-33088-monolith.dev-eks.codecademy.com/) |

testing in just monolith should be enough
monolith PR has screenshots as well

1. Visit preview env
2. Navigate global header w/ voice over
3. Open the resources menu and confirm that view all docs and view all topics are not read w/ the right arrow symbol
4. Open the catalog menu and confirm that the img next to 'take our quiz' is not read, and neither is the right arrow symbol
5. Approve!




[DISC-898]: https://codecademy.atlassian.net/browse/DISC-898?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DISC-900]: https://codecademy.atlassian.net/browse/DISC-900?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
